src/mame/juku/juku.cpp

```diff
@@ -413,19 +528,22 @@ void juku_state::pio0_portc_w(uint8_t data)
 
 	for (int i = 2; i < 8; i++)
 		if (BIT(data, i) !=  BIT(m_prev_portc, i))
-			logerror("pio0: c%d = %d\n", i, BIT(data, i));
+			osd_printf_verbose("pio0: c%d = %d\n", i, BIT(data, i));
```

Comment:
Using `logerror` or the `LOG` macro from logmacro.h is preferred for things like this.  You should be able to find plenty of examples to code that’s been migrated to use logmacro.h for debug logging.